packages/splits-kit/storybook/constants/chains.ts

```diff
@@ -150,4 +151,11 @@ export const STORYBOOK_CHAIN_INFO: ChainInfo = {
       `https://shape-mainnet.g.alchemy.com/v2/${process.env.STORYBOOK_ALCHEMY_API_KEY}`,
     ],
   },
+  [worldchain.id]: {
+    ...CHAIN_INFO[worldchain.id],
+    viemChain: worldchain,
+    rpcUrls: [
+      `https://worldchain-mainnet.g.alchemy.com/v2/${process.env.STORYBOOK_ALCHEMY_API_KEY}`,
```

Comment:
did you enable worldchain in our alchemy account?

Reply:
yesssir